.github/workflows/nightly_release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Nightly Release
+
+on:
+  workflow_call:
+
+jobs:
+    releaseDate:   
+        runs-on: ubuntu-latest
+        outputs:
+            date: ${{steps.releaseDate.outputs.date}}
+        steps:
+        - name: output current date
+          id: releaseDate
+          run: echo "date=$(date +'%Y-%m-%d')" >> $GITHUB_OUTPUT
+    createBranch:     
+        uses: ./.github/workflows/create_js_branch.yaml
+        with:
+           version: nightly-$(date +'%Y%m%d')
```

Comment:
the branch should always be the same. I'm wondering if it always works if we make the `version` just "nightly".

Reply:
Sorry, missed that one. Changed it to `nightly`.